components/timezone/README.md

```diff
@@ -49,8 +49,7 @@ As with time zone IDs, there are two interchangeable forms:
 ICU4X uses the short form.
 
 Note: in ICU4X, "metazone" is one word and "time zone" is two words, except for this crate
-and module name, where "timezone" is used with no separators. See
-<https://github.com/unicode-org/icu4x/issues/2507>.
```

Comment:
Question: Why did you remove the link? Removing cross-references seems like a regression to me.